docs/content/config/advanced/maintenance/update-and-cleanup.md

```diff
@@ -23,19 +23,6 @@ For more details, see the [manual](https://containrrr.github.io/watchtower/)
 
 ## Automatic Cleanup
 
-When you are pulling new images in automatically, it would be nice to have them cleaned up as well. There is also a docker image for this: [`spotify/docker-gc`](https://hub.docker.com/r/spotify/docker-gc/).
+For cleanup you can use the [`--cleanup`](https://containrrr.github.io/watchtower/arguments/#cleanup) option provided by `containrrr/watchtower`.
```

Comment:
Might as well keep some context about the intent, along with a compose example:

````suggestion
The service [`containrrr/watchtower`][watchtower] can monitor your images and automatically update them. Enable the [`--cleanup` option][watchtower-cleanup] to remove orphaned images from storage that are no longer used.

[watchtower]: https://containrrr.dev/watchtower/
[watchtower-cleanup]: https://containrrr.github.io/watchtower/arguments/#cleanup

```yaml
services:
  watchtower:
    image: containrrr/watchtower
    environment:
      - WATCHTOWER_CLEANUP=true
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```

!!! tip "Only update specific containers"

    The default `watchtower` service will poll every 24 hours for all images to update, not just within your `compose.yaml`.

    View their docs to provide a list of containers names and other settings if you'd like to restrict that behaviour.
````